testing/rabbitmq/rabbitmq.initd

```diff
@@ -0,0 +1,38 @@
+#!/sbin/openrc-run
+
+pidfile=/var/run/rabbitmq/rabbitmq.pid
```

Comment:
Alpine, as many others, stores pidfiles in the directory `/run` that is mounted as tmpfs. `/var/run` still exists, but it’s just a symlink to `/run` for backward compatibility.